cli/build.rs

```diff
@@ -154,7 +154,7 @@ mod ts {
     op_crate_libs.insert("deno.websocket", deno_websocket::get_declaration());
     op_crate_libs.insert("deno.webstorage", deno_webstorage::get_declaration());
     op_crate_libs.insert("deno.canvas", deno_canvas::get_declaration());
-    op_crate_libs.insert("deno.crypto", deno_crypto::get_declaration());
+    op_crate_libs.insert("deno.crypto", deno_crypto_get_declaration());
```

Comment:
Not sure about this one.

Could we just add a cfg here? Why do we insert these APIs if we don't support them?

Reply:
Types depend between each other and are referenced from the `lib.deno.shared_global.d.ts`. This is only inserted for the `deno (bin)` btw. denort does not embed any tsc related stuff.